drf_extra_fields/compat.py

```diff
@@ -0,0 +1,9 @@
+try:
+    from django.contrib.postgres.fields import FloatRangeField
+except ImportError:
+    FloatRangeField = None
+
+try:
+    from django.contrib.postgres.fields import DecimalRangeField
+except ImportError:
+    DecimalRangeField = None
```

Comment:
Do we need to check the compatibility of `DecimalRangeField`? Looks like it is added in Django2.2 and this package supports Django2.2 and newer versions.

Reply:
@cobang You are right.